index.html

```diff
@@ -9,85 +9,85 @@
 		<h1>SlideShow</h1>
 		<p class="intro">This Slide Show have the purpose of practicing my skill using HTML, CSS, and JavaScript. </p>
 	</div>
-	<div class="slideshow-container">
-		<div class="slides-container">
+	<div class="slideshow">
+		<div class="slidesContainer">
```

Comment:
`slidesContainer` this one could stay the same `slides-container` to avoir camelCase